Index: src/test/java/com/library/runner/CukesRunner.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.library.runner;\r\n\r\nimport io.cucumber.junit.Cucumber;\r\nimport io.cucumber.junit.CucumberOptions;\r\nimport org.junit.runner.RunWith;\r\n\r\n@RunWith(Cucumber.class)\r\n@CucumberOptions(\r\n\r\n        plugin = {\r\n                \"html:target/cucumber-report.html\",\r\n                \"rerun:target/rerun.txt\" ,\r\n                \"me.jvt.cucumber.report.PrettyReports:target/cucumber\"\r\n        },\r\n        features = \"src/test/resources/features\" ,\r\n        glue = \"com/library/steps\",\r\n        dryRun = true,\r\n        tags = \"@wip\"\r\n)\r\npublic class CukesRunner {\r\n\r\n}\r\n\r\n\r\n\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/test/java/com/library/runner/CukesRunner.java b/src/test/java/com/library/runner/CukesRunner.java
--- a/src/test/java/com/library/runner/CukesRunner.java	(revision 46a9762408a46d965824beb32611b690af0cdc3a)
+++ b/src/test/java/com/library/runner/CukesRunner.java	(date 1679967717755)
@@ -14,8 +14,8 @@
         },
         features = "src/test/resources/features" ,
         glue = "com/library/steps",
-        dryRun = true,
-        tags = "@wip"
+        dryRun = false,
+        tags = "@db"
 )
 public class CukesRunner {
 
